apps/web/src/components/Router.tsx

```diff
@@ -626,6 +626,14 @@ const UpdateSkillPage = React.lazy(() =>
       ),
   ),
 );
+const SkillLibraryPage = React.lazy(() =>
```

Comment:
Uhh, that is a good question. We actually have two possible places that link could go and there was no acceptance criteria about that. 🤔 

@substrae Is this link for the library or the showcase?

Reply:
Will need to be fixed definitely, it is one link right now

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/fa98c274-881f-43f0-8b94-19dad94c933f)

Reply:
Yeah, I remember building this hero and not having the skill link.  The solution was to combine them until the page existed.  Is this the page existing now?

Reply:
I guess it depends on that "this page" is. We have Skill Library and Skill Showcase, I'm not sure which one we should point this to since it just says "Skills" even though both pages show skills on them.

Reply:
@esizer 

The subtitle links to the library.

Reply:
Thanks, updated as part of [update dashboard hero skills link](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7831/commits/5b9f31eaa71d47c439c58b7a99d61c2421093754)

Reply:
Careful, you've got two a3's.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/a45a0078-ae89-421a-afcc-140384f05cce)

Reply:
🤦‍♀️ Thankfully, you have working eyes 👀 

Fixed in [fix final link in hero](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7831/commits/b55a6ab125b4ea8285efc7a2fe0a2fa6fe1774ce)